packages/core/contracts/Escrow.sol

```diff
@@ -36,8 +36,8 @@ contract Escrow is IEscrow, ReentrancyGuard {
     address public launcher;
     address payable public canceler;
 
```

Comment:
Add    ` uint32 private constant BULK_MAX_COUNT = 100` here to have variable packing